definitions/migrations/1.2.56/neftydrops.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE neftydrops_drops ADD COLUMN IF NOT EXISTS current_claimed bigint NOT NULL DEFAULT 0;
+
+UPDATE neftydrops_drops
+SET current_claimed = drop_stats.claims
```

Comment:
it may read better with a `currently_claimed` or `current_claims`, something feels fishy with the `current_claimed`

Reply:
We have the same field in the drops contract 😅